frontend-web/webclient/app/Applications/Studio/Applications.tsx

```diff
@@ -212,18 +161,27 @@ export const App: React.FunctionComponent = () => {
     const projectEntityField = React.useRef<HTMLInputElement>(null);
     const groupEntityField = React.useRef<HTMLInputElement>(null);
 
+    const groupOptions = React.useMemo(() => {
+        return allGroups.map((appGroup) => ({
+            text: appGroup.specification.title,
+            value: appGroup.metadata.id
+        })).sort((a, b) => a.text > b.text ? 1 : -1)
```

Comment:
I don't _really_ understand comparisons for arrays, but this should be similar.

```suggestion
        })).sort((a, b) => a.localeCompare(b))
```